search.go

```diff
@@ -65,6 +66,13 @@ func searchHandler(packagesBasePath string, cacheTime time.Duration) func(w http
 					internal, _ = strconv.ParseBool(v)
 				}
 			}
+
+			if v := query.Get("experimental"); v != "" {
+				if v != "" {
+					// In case of error, keep it false
+					experimental, _ = strconv.ParseBool(v)
```

Comment:
I'm not fan of swallowing errors... this should reflect in HTTP 400 Bad Request.

Reply:
You are right, if wrong params are submitted, we should just error and not assume defaults like we do here. As this applies to also the other api params, ok if I follow up with it?

Reply:
Sounds good!